declarations/FREE NOW.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "FREE NOW",
+  "documents": {
+    "Terms of Service": {
```

Indeed, like many of the documents to be added in the P2B instance (platforms that have at least a service component for other businesses, marketplaces…), this document does not have an `audience` of the `end user` of FREE NOW, a mobility service (that would be the person who gets driven somewhere), but an audience of the `business partners` of FREE NOW, the people who offer the rides. Thus:
```suggestion
    "Commercial Terms": {
```